pages/pipelines/hosted_agents/mac.md

```diff
@@ -35,8 +35,13 @@ Updated Xcode versions will be available one week after Apple offers them for do
 
 ### Xcode
 
-- Xcode 14.3.1
-- Xcode 15.2
+- Xcode 15.4 (15F5021i)
+- Xcode 15.4 (15F31d)
+- Xcode 15.3 (15E204a)
+- Xcode 15.3 (15E204a)
+- Xcode 15.2 (15C500b)
+- Xcode 15.1 (15C65)
+- Xcode 14.3.1 (14E300c)
```

Comment:
What do the alphanumeric values in brackets mean?

Reply:
Added wording to show this is the Xcode `Build` version. Let me know if it's still needs more clarity
This comes form https://xcodereleases.com/

Reply:
Thanks for your update @dimiexplorer and for clarifying this. I suspect we won't need to specify this detail in the docs and the readers can make this assumption based on the additional edits you've made. Nevertheless, we have this detail captured in this PR.

Anyway, I'll approve this PR and it's ready for you to merge in!